analysis/parser/RawTextParser/src/main/kotlin/de/maibornwolff/codecharta/parser/rawtextparser/ProjectMetrics.kt

```diff
@@ -0,0 +1,12 @@
+package de.maibornwolff.codecharta.parser.rawtextparser
+
+import java.util.concurrent.ConcurrentHashMap
+
+class ProjectMetrics {
+
+    val metricsMap = ConcurrentHashMap<String, FileMetrics>()
```

Comment:
I think it is not necessary to expose this.
Methods should be offered in this class for operations running on metricsMap.

Reply:
Since the metricsMap is used in a coroutines context, it will always be implemented using a ConcurrentHashMap. Adding a getter might make the code longer without much payoff.